src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -213,6 +213,11 @@ class SqlKernel extends Disposable implements nb.IKernel {
 		}
 	}
 
+	// Gets language info depending on connection provider.
+	private getLanguageFlavor(providerName: string): string {
+		return providerName === 'KUSTO' ? 'kusto' : 'sql';
```

Comment:
probably should avoid special-cases for KUSTO provider.  we're still in the process of trying to remove all the MSSQL special cases.